### ISSUE TITLE
[WCAG] Update 'aria-label' props to always use the explicit translated string

### ISSUE THREAD

**Issue by kenchase** [Canadian-Geospatial-Platform/GeoView#3590]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

Re: PR #3558 - Related to icon-button implementation. Specifically, how Drawer was implemented to display translation text, rather than translation keys

- packages/geoview-drawer/src/index.tsx
- packages/geoview-core/src/ui/icon-button/icon-button.tsx

Comment from Alex...

That seems like something we should be standardizing across the codebase, and I'd argue we should have the 'aria-label' props always use the explicit translated string (the 't' call being done) rather than expecting the sub-component to do the 't()' call and have the caller send a 'guessed' translation key string to sub-components.

It seems more intuitive to me to work with explicit translated strings all the time? In any case, I feel like mix-matching the 'aria-label' property expected parameter (sometimes implicit and sometimes explicit) might get confusing.

### Expected Behavior

The 'aria-label' props should always use the explicit translated string (the 't' call being done) rather than expecting the sub-component to do the 't()' call
